README.md

```diff
@@ -12,96 +12,96 @@ Pull requests are welcome. Please keep the list in alphabetical order.
-[DataRobot](http://www.datarobot.com)             | Data science as a service                | Python, R, Web, data stuff                     | ?                                    | ?                       | :keyboard: [Jobs](http://www.datarobot.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-DataRobot-EI_IE911432.11,20.htm)
-[DataStax](http://www.datastax.com/)              | Enterprise Cassandra                     | Cassandra, Java, DevOps                        | ✓                                    | ?                       | :keyboard: [Jobs](http://www.datastax.com/company/careers)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-DataStax-EI_IE524413.11,19.htm)
-[DigitalOcean](https://www.digitalocean.com/)     | Infrastracture as a service              | C++, Go, Ruby, Perl, Web                       | ✓                                    | ?                       | :keyboard: [Jobs](https://www.digitalocean.com/company/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-DigitalOcean-EI_IE823482.11,23.htm)
-[Doist](https://doist.com)                        | Productivity and team communication apps | Web, Mobile                                    | ✓                                    | ?                       | :keyboard: [Jobs](https://doist.com/jobs/)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Doist-EI_IE1102553.11,16.htm)
-[Dribbble](https://dribbble.com)                  | Where Designers get Inspired & Hired     | Ruby, Web, Mobile                              | ✓                                    | ?                       | :keyboard: [Jobs](https://dribbble.com/careers)
+[ConsenSys](https://consensys.net/)               | Blockchain studio                        | Many (see job page)                            | ?                                    | Anywhere                | :keyboard: [Jobs](https://consensys.net/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com/Reviews/ConsenSys-Reviews-E1613426.htm)
+[Crossover](https://www.crossover.com/)           | Full-time long-term projects             | Many (see job page)                            | ✗                                    | Depends - this is a job aggregator | :keyboard: [Jobs](https://app.crossover.com/x/marketplace/available-jobs)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Crossover-for-Work-EI_IE1041838.11,29.htm)
+[DataDog](http://www.datadoghq.com)               | Monitoring & analytics                   | Scala, Java, Python, Go                        | ✓                                    | Only small % of positions are remote | :keyboard: [Jobs](https://www.datadoghq.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Datadog-EI_IE762009.11,18.htm)
```

Comment:
It looks like there are quite a few remote engineering jobs, but in any case, this column should be about the locations available rather than the percentage of positions. :slightly_smiling_face:

Reply:
👍